packages/component-library/package.json

```diff
@@ -27,7 +27,9 @@
     "test:watch": "stencil test --spec --e2e --watchAll",
     "generate": "node generate-component.js",
     "storybook": "start-storybook -p 6006 --no-dll -s ./www",
-    "build-storybook": "build-storybook && cp -r ./www/build/ ./storybook-static/"
+    "build-storybook": "build-storybook && cp -r ./www/build/ ./storybook-static/",
+    "download-icons": "node ./scripts/icons/downloadIcons.mjs",
```

Comment:
Please add some documentation, in the Readme. About how to use these scripts, when to use them and what they achieve.